src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -472,7 +474,8 @@ PARAM_DEFINE_FLOAT(FW_FLAPS_SCL, 1.0f);
 /**
  * Flaps setting during take-off
  *
- * Sets a fraction of full flaps (FW_FLAPS_SCL) during take-off
+ * Sets a fraction of full flaps (FW_FLAPS_SCL) during take-off.
```

Comment:
oo thanks for adding the clarification on this param.. i actually misunderstood it in the last weeks! thought it was percentage of full. not a fraction of the already scaled by `FW_FLAPS_SCL` param... 

which actually leads me to another thought.. why? should we just go ahead and change this now to mean fraction of 100%? wouldn't this be much easier to understand? -- same for landing scale. Or maybe I'm missing a relevant justification for the "scale of the scale" approach.

Reply:
I didn't want to change everything at once:) Here it really should be about enabling spoiler and flaps for VTOLs, let's follow up with another PR for more refactoring. 
I though 100% agree with you though, I would remove `FW_FLAPS_SCL` completely.

Reply:
Changed my mind, here it is: https://github.com/PX4/PX4-Autopilot/pull/19329/commits/3689864c2a2a6c8bb0fd2edede65ebc74fe61c1a